nav_msgs/OccupancyGrid originalMap
nav_msgs/OccupancyGrid cleanMap
#it's the equivalent of an array of pairs [(dir1_x, dir1_y), (dir2_x, dir2_y), ...]
float32[] directions
